onnxruntime/python/onnxruntime_inference_collection.py

```diff
@@ -226,6 +226,36 @@ def run(self, output_names, input_feed, run_options=None):
                 return self._sess.run(output_names, input_feed, run_options)
             raise
 
+    def run_async(self, output_names, input_feed, callback, user_data, run_options=None):
+        """
+        Compute the predictions asynchronously in a separate cxx thread from ort intro-op threadpool.
```

Comment:
```suggestion
        Computes the predictions asynchronously in a separate thread belonging to the ORT intra-op threadpool.
```

Reply:
The "Compute" keeps the style in the same doc, and prefer to keep the "from".